WireAPI/Sources/WireAPI/APIs/ConversationsAPI/ConversationsAPIV2.swift

```diff
@@ -18,10 +18,6 @@
 
 import Foundation
 
-public protocol APIBuilder {
-    associatedtype API
-
-    init(httpClient: any HTTPClient)
-
-    func makeAPI(for version: APIVersion) -> API
+class ConversationsAPIV2: ConversationsAPIV1 {
+    override var apiVersion: APIVersion { .v2 }
 }
```

Comment:
was the only need of this to make it work on snaphot tests?

Reply:
I thought the pattern is like this, that we create one class for every version? cc @typfel 
From my point of view we can remove all classes, that didn't make any changes...
e.g. if you have changes in V6 it can be `ConversationsAPIV6: ConversationsAPIV1` and the `ConversationsAPIBuilder` takes care of the right instantiation?